rust_src/crates/ng_async/src/lib.rs

```diff
@@ -0,0 +1,24 @@
+#![allow(non_camel_case_types, non_snake_case, non_upper_case_globals)]
+#![feature(concat_idents)]
+#![feature(async_closure)]
+
+extern crate libc;
+
+extern crate futures;
+extern crate lazy_static;
```

Comment:
I think we don't need most of those `extern crate`s in 2018 edition.

Reply:
:+1: